worker/src/handlers/statement/allowedWorkerQueues.js

```diff
@@ -0,0 +1,51 @@
+import logger from 'lib/logger';
+import {
+  STATEMENT_QUERYBUILDERCACHE_QUEUE,
+  STATEMENT_EXTRACT_PERSONAS_QUEUE,
+  STATEMENT_FORWARDING_QUEUE,
+} from 'lib/constants/statements';
+
+/**
+ * allowable Worker Queues
+ *
+ * @type {string[]}
+ */
+const allowableWorkerQueues = [
+  STATEMENT_QUERYBUILDERCACHE_QUEUE,
+  STATEMENT_EXTRACT_PERSONAS_QUEUE,
+  STATEMENT_FORWARDING_QUEUE,
+];
+
+/**
+ * @param {string | undefined} allowedWorkerQueuesString
+ * @return {string[]}
+ * @throws {Error}
+ */
+const getAllowedWorkerQueues = (allowedWorkerQueuesString) => {
+  if (allowedWorkerQueuesString === undefined) {
+    return allowableWorkerQueues;
+  }
+
+  if (allowedWorkerQueuesString === '') {
+    return [];
+  }
+
+  return allowedWorkerQueuesString.split(',').reduce(
```

Comment:
We could technically use `filter` instead of `reduce` here, but I'm not too worried.

Reply:
Exactly! I'll rewrite here with filter.

Reply:
Okay thanks Hota 👍